File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1066.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

pos=0
neg=0
par=0
imp=0

for(i=0;i<5;i++){
    a = parseInt(lines.shift());
    if(a>0){
        pos++;
    } else if(a<0){
        neg++;
    }
    if(a%2==0){
        par++;
    } else{
        imp++;
    }
}
console.log(par+" valor(es) par(es)");
console.log(imp+" valor(es) impar(es)");
console.log(pos+" valor(es) positivo(s)");
console.log(neg+" valor(es) negativo(s)");

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1066.py
pos=0
neg=0
par=0
imp=0

for i in range(0,5):
    a = int(input())
    if(a>0):
        pos=pos+1
    elif(a<0):
        neg=neg+1
    if(a%2==0):
        par=par+1
    else:
        imp=imp+1

print("{} valor(es) par(es)".format(par))
print("{} valor(es) impar(es)".format(imp))
print("{} valor(es) positivo(s)".format(pos))
print("{} valor(es) negativo(s)".format(neg))
